languagetool-language-modules/fr/src/main/resources/org/languagetool/rules/fr/grammar.xml

```diff
@@ -117557,6 +117557,14 @@ Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301, USA
                     <token postag="A" min="0" max="3"/>
                     <token regexp="yes">à|aux?</token>
                 </antipattern>
+                <antipattern>
+                    <token postag="D.*" postag_regexp="yes"/>
+                    <token postag="N.*" postag_regexp="yes"/>
+                    <token regexp="yes">et</token>
+                    <token postag="D.*" postag_regexp="yes"/>
+                    <token postag="N.*" postag_regexp="yes"/>
+                    <token regexp="yes">diff?èrent$</token>
```

Comment:
* No example to very the antipattern?
* why "è" instead of "é" here?
* the "$" is useless.  Did you mean "différente?s"?

Reply:
Hi! 
Example added [here ](https://github.com/languagetool-org/languagetool/pull/7688).
I was matching the pattern that generates OT (<token regexp="yes">diff?èrent$|adhèrent$|réfèrent$) but it's actually not needed.
Thanks :)